new update for Fächer and Genres

Michi135 opened the pull request:
Lehramtfach is now a String and therefore freely changeable. And I added  Angewandte Informatik. Added some new Genres for Unikino which should be useable in the Strapi.

Michi135 committed:
new update for Technomathe and Genres

Lehramtfach is now a String and therefore freely changeable. And I added  Angewandte Informatik.
Added some new Genres for Unikino which should be useable in the Strapi.

Olli1080 commented:
You could do this if you want but keep in mind that this will break translation for unknown string values on the frontend

Olli1080 commented:
Bump @crimeZeT

Michi135 committed:
Merge remote-tracking branch 'upstream/vite-migration' into miTestBranch

Michi135 committed:
Revert "new update for Technomathe and Genres"

This reverts commit 3ec3cfb83ef2bcdbdb8cd979ff7842ee31b35fc8.

Michi135 committed:
Lehramt Vertreter improvements

Changed zweitfach and drittfach in strings that are inserted via Strapi in EN or in DE. Therefore not all combinations are needed to be implemented hardcoded.

Michi135 committed:
commented the friday Sprechstunde

Olli1080 committed:
Review and package updates
